.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -29,15 +31,18 @@ about: "If you're sure it's reproducible and not just your machine: submit an is
   </pre>
 </details>
 
-## What you expected to happen
+## Expected behavior
+_What you expected to happen._
 
 <!-- replace me -->
 
-## Step-by-step reproduction instructions (by running `brew` commands)
+## Steps to reproduce
+_Step-by-step reproduction instructions (by running `brew` commands)_
```

Comment:
Pretty minor but I noticed all of the other italicized descriptions end in a period. I think it's up to you if you want to add it for consistency. I have no problem keeping it this way

Reply:
Thanks, Rylan. Added and force-pushed.